src/components/NavBar/nav.css

```diff
@@ -0,0 +1,11 @@
+.navbar {
```

Comment:
I would prefer to keep all of our CSS in `index.css`.  I'd prefer not to have a bunch of css files especially if we are mostly using bootstrap

Reply:
Okay, I'll make that change. Wasn't sure best design choice for this in terms of modularity if we should keep components having their own css files, I will make the change